tests/ips-state-1/README.md

```diff
@@ -5,9 +5,7 @@ be full passed with no alerts, while the TLS flow should be dropped.
 
 ## Current Observations
 
-- HTTP response packets are being logged as dropped, however the transaction is
-  logged suggesting the drop is only in logging only, but not actually
-  occurring.
+- HTTP flows are logged as `passed`, as expected.
 
-- All the TLS packets apear to be getting dropped, but `flow.action` is never
-  set to true.
+- All the TLS packets appear to be getting dropped, but `flow.action` is never
+  set to drop.
```

Comment:
Is there a ticket reference to fix this ?

Reply:
https://redmine.openinfosecfoundation.org/issues/6976

Might not be a real issue, addressing the comment on https://github.com/OISF/suricata-verify/pull/1975 and submitting a new PR will hopefully clarify that.

Reply:
So, should this PR be a draft until the clarification ?

Reply:
Fair point.

Reply:
Discussed with Victor, bug confirmed (ticket https://redmine.openinfosecfoundation.org/issues/6976)

Reply:
So, should CI be red if there is a bug to fix ?

Reply:
I've kept that check for a different test, and left this one with the update, only - https://github.com/OISF/suricata-verify/pull/1987